packages/ckeditor5-build-markdown/src/ckeditor.js

```diff
@@ -99,6 +102,21 @@ MarkdownEditor.defaultConfig = {
 			'mergeTableCells'
 		]
 	},
+	dropdownlist: [
+		{ label: 'Account Name', value: '{{account.name}}' },
+		{ label: 'Sender Email', value: '{{template.senderEmailAddress}}' },
+		{ label: 'Sender Name', value: '{{template.senderName}}' },
```

Comment:
These two values need to be excluded from this list for the phone script editor... Not sure how best to accomplish that.